src/main/resources/data/1817/Map.xml

```diff
@@ -0,0 +1,111 @@
+<Map tileOrientation="EW" letterOrientation="vertical" even="A">
```

Comment:
I don't see Indianapolis, Louisville, or Blacksburg in the map. I suspect this map isn't the 1817 map.